app/controllers/users_controller.rb

```diff
@@ -9,7 +9,7 @@ def new
   def create
     @user = User.new(user_params)
     if @user.save
-      session[:user_id] = @user.id
+      UserMailer.welcome_email(@user).deliver_now!
```

Comment:
@jynti Should this be in the controller? Why have we used deliver_now! and not deliver_now?